prow/plugins.yaml

```diff
@@ -129,6 +129,11 @@ blockades:
   - ^events/2016
   - ^events/elections/2017
   explanation: "These files are historical, and from events that have already occurred."
+- repos:
+  - kubernetes/sig-release
```

Comment:
the plugin also needs to be enabled for `kubernetes/sig-release`

Reply:
@BenTheElder -- fixed it (I think)!